cl/settings/project/microservices.py

```diff
@@ -4,6 +4,10 @@
 
 DISCLOSURE_HOST = env("DISCLOSURE_HOST", default="http://cl-disclosures:5050")
 DOCTOR_HOST = env("DOCTOR_HOST", default="http://cl-doctor:5050")
+INCEPTION_HOST = env(
+    "INCEPTION_HOST", default="http://host.docker.internal:8005"
```

Comment:
This is a bit weird. Don't we just want to call it:

```suggestion
    "INCEPTION_HOST", default="http://cl-inception:8005"
```

Reply:
We'll need it as part of the compose file since we'll be adding semantic search, which will use it to vectorize queries, so we might as well add it to this PR.

Reply:
I remember we were worried about the footprint Inception would add to our dev Docker environment, but it makes sense to include it for local work on semantic search.

I've added Inception to the docker-compose setup. Initially, I encountered disk space issues on my dev env, so I had to increase the disk limit in Docker Desktop due to the image is greater than 8GB.

Now that I've pushed the changes, we can see the same issue occurring in the GitHub CI.

![Screenshot 2025-04-11 at 1 13 16 p m](https://github.com/user-attachments/assets/8b3a7322-63c8-460c-9f5d-9f9787c0ea8f)

Is there a way to increase the disk space of the Github CI instance?

Reply:
Oh, right, I remember now. I don't think there's a way to do this except to host our own Github Runners. Let's yank it for now and I guess we'll be OK.

Reply:
Removed